modules/ROOT/pages/tuning-domains.adoc

```diff
@@ -0,0 +1,16 @@
+= Domains
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Domains use greatly enhance performance when you deploy multiple services on the same on-premises Mule runtime engine (Mule). By providing a centralized point for all the shared resources, domains make the class-loading process (and, therefore, metaspace memory usage) more efficient, especially because domain dependencies declared in the `pom.xml` file are also shared in the domain apps.
```

Comment:
**Domains use greatly enhance performance**, this sentence causes a bit of confusion for me when I read it, I'd rewrite it like: 
**Using domains greatly enhances performance**...